src/tribler/ui/src/dialogs/SaveAs.tsx

```diff
@@ -155,6 +158,8 @@ export default function SaveAs(props: SaveAsProps & JSX.IntrinsicAttributes & Di
                 setError(`${t("ToastErrorGetMetainfo")} ${t("ToastErrorGenNetworkErr")}`);
             } else if (isErrorDict(response)) {
                 setError(`${t("ToastErrorGetMetainfo")} ${response.error.message}`);
+                // Disable for all errors, except metainfo errors (timeouts)
+                setDisabled(response.error.message !== "metainfo error");
```

Comment:
This is not enough to revert #8403, it reintroduces #8403 for slow loading sites. For example, entering `https://www.wish.com/` and clicking `Download`, leads to:

![](https://github.com/user-attachments/assets/222f6c55-5579-4089-bc27-fa69a1d54d89)

I appreciate your cause of "removing any feedback that something is happening", but this is not sufficient. Please either fix your fix, undo your unfix, or add a spinner or loadbar? I'm O.K. with anything but breaking it again.

Reply:
OK, I've removed my fix.